articles/client-platforms/_includes/_mfa-login.md

```diff
@@ -0,0 +1,5 @@
+There is no need to update the code we have created in [Login](/docs/quickstart/spa/react/01-login) tutorial. As soon you try to sign in, you should be prompted to setup an 2-Factor authenticator application (auth0 Guardian is the default).
```

Comment:
@alemata Change `setup an 2-Factor` to `setup a second factor`
Capitalize Auth0 (`auth0 Guardian` -> `Auth0 Guardian`)